server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileRepository.java

```diff
@@ -15,37 +15,40 @@
+            "PGP_SYM_DECRYPT(cast(workEmail as bytea), '${aes.key}') as workEmail, " +
             "employeeId, startDate, " +
-            "PGP_SYM_DECRYPT(cast(bioText as bytea),'${aes.key}') as bioText, " +
+            "PGP_SYM_DECRYPT(cast(bioText as bytea), '${aes.key}') as bioText, " +
             "supervisorid, terminationDate " +
             "FROM member_profile mp " +
             "WHERE  (:workEmail IS NULL OR PGP_SYM_DECRYPT(cast(mp.workEmail as bytea), '${aes.key}') = :workEmail) ",
-             nativeQuery = true )
+            nativeQuery = true)
     Optional<MemberProfile> findByWorkEmail(@NotNull String workEmail);
 
     @Query(value = "SELECT id, " +
-            "PGP_SYM_DECRYPT(cast(mp.name as bytea),'${aes.key}') as name, " +
-            "PGP_SYM_DECRYPT(cast(title as bytea),'${aes.key}') as title, " +
+            "PGP_SYM_DECRYPT(cast(mp.name as bytea), '${aes.key}') as name, " +
+            "PGP_SYM_DECRYPT(cast(title as bytea), '${aes.key}') as title, " +
             "pdlid, " +
-            "PGP_SYM_DECRYPT(cast(location as bytea),'${aes.key}') as location, " +
-            "PGP_SYM_DECRYPT(cast(workEmail as bytea),'${aes.key}') as workEmail, " +
+            "PGP_SYM_DECRYPT(cast(location as bytea), '${aes.key}') as location, " +
+            "PGP_SYM_DECRYPT(cast(workEmail as bytea), '${aes.key}') as workEmail, " +
             "employeeId, startDate, " +
-            "PGP_SYM_DECRYPT(cast(bioText as bytea),'${aes.key}') as bioText, " +
+            "PGP_SYM_DECRYPT(cast(bioText as bytea), '${aes.key}') as bioText, " +
             "supervisorid, terminationDate " +
             "FROM member_profile mp " +
             "WHERE (:name IS NULL OR PGP_SYM_DECRYPT(cast(mp.name as bytea), '${aes.key}') = :name) " +
             "AND (:title IS NULL OR PGP_SYM_DECRYPT(cast(mp.title as bytea), '${aes.key}') = :title) " +
             "AND (:pdlId IS NULL OR mp.pdlId = :pdlId) " +
             "AND (:workEmail IS NULL OR PGP_SYM_DECRYPT(cast(mp.workEmail as bytea), '${aes.key}') = :workEmail) " +
-            "AND (:supervisorId IS NULL OR mp.supervisorId = :supervisorId) ", nativeQuery = true )
-    List<MemberProfile> search(@Nullable String name, @Nullable String title, @Nullable String pdlId, @Nullable String workEmail, @Nullable String supervisorId);
+            "AND (:supervisorId IS NULL OR mp.supervisorId = :supervisorId) ", nativeQuery = true)
+    List<MemberProfile> search(@Nullable String name, @Nullable String title, @Nullable String pdlId,
+                               @Nullable String workEmail, @Nullable String supervisorId);
 
     List<MemberProfile> findAll();
 
-    String findNameById(@NotNull UUID id);
+    @Query(value = "SELECT PGP_SYM_DECRYPT(cast(mp.name as bytea), '${aes.key}') FROM member_profile as mp " +
+            "WHERE mp.id = :id", nativeQuery = true)
+    String findNameById(@NotNull String id);
```

Comment:
rather than changing this to String, I think I would have added an explicit cast to string in the where clause. this would be consistent with what is happening with the type hints in the entity objects.

Reply:
The `id` column of the member_profile table is of type `varchar`. If I keep `UUID` as the method's parameter type, I would need to convert `UUID` to a string. How would you do that conversion in the query?

Reply:
`WHERE mp.id = cast(:id as varchar)` I believe

Reply:
Okay, I'll try that.